grammar.js

```diff
@@ -705,7 +705,8 @@ module.exports = grammar({
         $.identifier,
         $.parenthesized_expression,
       )),
-      choice('+=', '-=', '*=', '/=', '%=', '^=', '&=', '|=', '>>=', '>>>=', '<<=', '**='),
+      choice('+=', '-=', '*=', '/=', '%=', '^=', '&=', '|=', '>>=', '>>>=',
+             '<<=', '**=', '&&=', '||=', '??='),
```

Comment:
I like to cut lines at 80 characters. I could do differently if you want (or use an auto-format tool).